config/config.toml

```diff
@@ -154,10 +154,12 @@ kafkaDialTimeoutInSec = 10
 retryTimeWindowInMins = 180
 minRetryBackoffInS = 10
 maxRetryBackoffInS = 300
+workerRequestBatchTimeoutInS = 5
```

Comment:
We need one more configuration for MaxBatchSize. And a boolean flag for enabling batching.

Also, I think we can remove the worker prefix.